doc/hiop-policy-compatibility.md

```diff
@@ -0,0 +1,47 @@
+# xSDK Community Policy Compatibility for HiOp
+
+This document summarizes the efforts of current and future xSDK member packages to achieve compatibility with the xSDK community policies. Below only short descriptions of each policy are provided. The full description is available [here](https://github.com/xsdk-project/xsdk-community-policies)
+and should be considered when filling out this form.
+
+Please, provide information on your compability status for each mandatory policy, and if possible also for recommended policies.
```

Comment:
Hello, I was reading this document (I'm a silent viewer of these repos) and I found just one typo :)
```suggestion
Please, provide information on your compatibility status for each mandatory policy, and if possible also for recommended policies.
```

Reply:
thanks